docs/pages/blog/introducing-pigment-css.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'Introducing Pigment CSS: the next generation of CSS-in-JS'
+description: 'Pigment CSS offers significant performance gains along with RSC and App Router support.'
+date: 2024-05-15T00:00:00.000Z
+authors: ['samuelsycamore']
```

Comment:
I think we should have @brijeshb42 and @siriwatknp as authors too. If feels wrong to introduce a product without their acting product leads, I mean, I can't think of a successful project that wasn't released/first introduced this way.